src/internet_identity/src/storage.rs

```diff
@@ -106,7 +159,7 @@ impl<T: candid::CandidType + serde::de::DeserializeOwned> Storage<T> {
                 &mut header as *mut _ as *mut u8,
                 std::mem::size_of::<Header>(),
             );
-            stable64_read(0, slice);
```

Comment:
out of curiosity: is the address size abstracted away now or do we pass that in as configuration somewhere?

Reply:
`stable_structures` supports 64bit addresses only. Which has the nice benefit that now one cannot get it wrong accidentally. 🙂